src/websocket/app.ts

```diff
@@ -36,7 +36,7 @@ export class AppWebsocket implements AppApi {
 
   _requester = <ReqO, ReqI, ResI, ResO>(tag: string, transformer?: Transformer<ReqO, ReqI, ResI, ResO>) =>
     requesterTransformer(
-      req => this.client.request(req).then(catchError),
+      (req, timeout) => promiseTimeout(this.client.request(req), timeout).then(catchError),
```

Comment:
I don't remember what `tag` is here, but if it has helpful context you could pass it into `promiseTimeout` to include in the timeout error message